fields.go

```diff
@@ -36,3 +36,14 @@ func parseFields(fields string) (FieldSet, error) {
 	}
 	return FieldSet{}, ErrInvalidFields{fields}
 }
+
+func parseExtraSymbols(symbols string) (FieldSet, error) {
+	symbolsPattern := regexp.MustCompile(`^\+q$`)
+	if symbols == "" {
+		return FieldSet{}, nil
+	}
+	if symbolsPattern.MatchString(symbols) {
+		return FieldSet{Receiver: true, Module: true}, nil
```

Comment:
There's no way to specify `Receiver` and `Module` separately, so let's merge them into a single field for now. We can call them `ExtraTags` or something like that.